composer.json

```diff
@@ -46,8 +46,6 @@
         "php-cs-fixer/accessible-object": "^1.1",
         "php-cs-fixer/phpunit-constraint-isidenticalstring": "^1.4",
         "php-cs-fixer/phpunit-constraint-xmlmatchesxsd": "^1.4",
-        "phpspec/prophecy": "^1.18",
```

Comment:
pls update 
```
tests/AutoReview/ProjectCodeTest.php:        $message = sprintf('Class %s must not use PHPUnit\'s mock,, it shall use ->prophesize() instead.', $testClassNa>
```

as with potential merge of this PR, the message in tests will get very confusing to contributors

Reply:
Right, the double comma would be very confusing, fixed.